Authenticator/Source/TokenListViewController.swift

```diff
@@ -138,11 +143,11 @@ extension TokenListViewController {
         return viewModel.rowModels.count
     }
 
-    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier(NSStringFromClass(TokenRowCell.self), forIndexPath: indexPath)
-        if let cell = cell as? TokenRowCell {
-            updateCell(cell, forRowAtIndexPath: indexPath)
-        }
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath)
+        -> UITableViewCell
+    {
+        let cell = tableView.dequeueReusableCellWithClass(TokenRowCell.self)
```

Comment:
Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)